### ISSUE TITLE
Release notes / changelog for 4.0 betas

### ISSUE THREAD

**Issue by radames** [keystonejs/keystone#4481]
Couldn't find a clear way to see the changelogs for thew v4.0 beta releases?
Is there any place where the new Beta versions are being tracked?
Wouldn't be great to have [HISTORY.md](https://github.com/keystonejs/keystone/blob/master/HISTORY.md) update also for major features and/or bugs on thew new beta versions?
Just saw the new v4.0.0-beta.7 tag, but it's hard to keep track of all diffs on the commits.

**Comment by MillerApps** [comment]
Second that.

**Comment by htor** [comment]
yes, it would be logical to do that. this is something https://github.com/keystonejs/keystone/pull/4476 would start. the documentation efforts are not very good yet, and it looks like to me it is not being prioritised here, so it remains unclear what happens next.

**Comment by stennie** [comment]
Agree there should be clearer change history for the 4.0 betas. Betas through `v4.0.0-beta5` already had release notes drafted in the [GitHub releases](https://github.com/keystonejs/keystone/releases) page so I've published those as a start.

This release note format differs from `HISTORY.md`, which includes highlights and contributor thanks rather than an exhaustive list of changes.

@htor Thanks for your PR and offer of assistance!  I've recently started helping @autoboxer get Keystone 4 back on track including opening up community/contributor discussion and identifying critical tasks to remove the beta label from Keystone 4. There is now an active [milestone for Keystone 4.0.0](https://github.com/keystonejs/keystone/milestone/5) release, and documentation is definitely on the shortlist for improvement. We're aiming to coordinate discussion via the KeystoneJS Slack which now has an [open invite page](https://launchpass.com/keystonejs). (I've sent you an invite, as well).

Regards,
Stennie
